layouts/default/readme.md

```diff
@@ -0,0 +1,704 @@
+└────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+```
+
+```
+LAYOUT_60_ansi_arrow
+┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴┬───┬───┤
+│        │   │   │   │   │   │   │   │   │   │      │   │   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴──┬┴──┬───┼───┼───┤
+│    │    │    │                        │   │   │   │   │   │
+└────┴────┴────┴────────────────────────┴───┴───┴───┴───┴───┘
+```
+
+```
+LAYOUT_60_ansi_split_bs_rshift
+┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+│   │   │   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │
+├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤
+│    │    │    │                        │    │    │    │    │
+└────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+```
+
+```
+LAYOUT_60_ansi_tsangan
+┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+│   │   │   │   │   │   │   │   │   │   │   │   │   │       │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────────┤
+│        │   │   │   │   │   │   │   │   │   │   │          │
+├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─────┤
+│     │   │     │                           │     │   │     │
+└─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘
+```
+
+```
+LAYOUT_60_hhkb
+┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+│   │   │   │   │   │   │   │   │   │   │   │   │   │   │   │
+├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+│     │   │   │   │   │   │   │   │   │   │   │   │   │     │
+├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴─────┤
+│      │   │   │   │   │   │   │   │   │   │   │   │        │
+├──────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┤
+│        │   │   │   │   │   │   │   │   │   │   │      │   │
+├─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─┴───┤
+│▒▒▒▒▒│   │     │                           │     │   │▒▒▒▒▒│
+└─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘
```

Comment:
```suggestion
└─────┬──┴┬──┴──┬┴───┴───┴───┴───┴───┴───┴──┬┴───┴┬───┬─┴───┘
      │   │     │                           │     │   │
      └───┴─────┴───────────────────────────┴─────┴───┘
```
?

Reply:
I actually had it this way originally, but then I thought it looked "off," somehow.

Reply:
I like having the non-rectangular shapes, it shows off what you can do with box drawing characters 😄